examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -10,6 +10,21 @@ filegroup(
     visibility = ["//:__subpackages__"],
```

Comment:
Note: this can also be done with a macro, but I preferred to keep the example simple.